scripts/approve_draft_registrations.py

```diff
@@ -0,0 +1,33 @@
+import sys
+import logging
+
+from website.app import init_app
+from website.models import DraftRegistration, Sanction, User
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.WARN)
+logging.disable(level=logging.INFO)
+
+
+def main(dry_run=True):
+    if dry_run:
+        logger.warn('DRY RUN mode')
+    pending_approval_drafts = DraftRegistration.find()
+    need_approval_drafts = [draft for draft in pending_approval_drafts
+                            if draft.requires_approval and draft.approval and draft.approval.state == Sanction.UNAPPROVED]
```

Comment:
It seems like this could be done more efficiently in a mongo query object,  rather than filtering the list of all results.

Eg (untested and guessing at default values)

`Q('requires_approval', 'eq', True) & Q('approval', 'ne', None)`

(not sure if approval.state can be directly queried this way; I'd need to dig through odm docs a bit more. References get weird in mongo)